src/wp-content/themes/twentytwentyfive/patterns/hidden-written-by.php

```diff
@@ -10,8 +10,8 @@
  */
 
 ?>
-<!-- wp:group {"style":{"spacing":{"blockGap":"0.2em","margin":{"bottom":"var:preset|spacing|60"}}},"textColor":"accent-4","fontSize":"small","layout":{"type":"flex","flexWrap":"wrap"}} -->
-<div class="wp-block-group has-accent-4-color has-text-color has-link-color has-small-font-size" style="margin-bottom:var(--wp--preset--spacing--60)">
+<!-- wp:group {"style":{"spacing":{"blockGap":"0.2em","margin":{"bottom":"var:preset|spacing|60"}}},"fontSize":"small","layout":{"type":"flex","flexWrap":"wrap"}} -->
+<div class="wp-block-group  has-text-color has-link-color has-small-font-size" style="margin-bottom:var(--wp--preset--spacing--60)">
```

Comment:
```suggestion
<div class="wp-block-group has-small-font-size" style="margin-bottom:var(--wp--preset--spacing--60)">
```

When the color is removed, all related classes should be removed, too. 

(After clearing the Text Color selection, I still needed to delete `has-link-color` from the Additional CSS Classes field.)